### DISCUSSION TITLE
Invalid forward email

### DISCUSSION THREAD

**Discussion by felinto-dev** [chatwoot/chatwoot#10288] [Q&A]
![image](https://github.com/user-attachments/assets/909ef3b6-3bcf-4fcb-8485-69fa8cef387a)

The environment variable 'MAILER_INBOUND_EMAIL_DOMAIN' is already set but still the generated forwarding email has the ending "@false". What do I do? I'm using caprover and chatwoot version v3.13.0.

**Comment by sojan-official** [comment]
duplicate : https://github.com/chatwoot/chatwoot/issues/10304
